index.js

```diff
@@ -0,0 +1,30 @@
+// import express (after npm install express)
+const express = require('express');
+
+// create new express app and save it as "app"
+const app = express();
+
+// server configuration
+const PORT = 3000;
+
+// Getting data from json files
+var packageData  = require('./package.json');
+
+// create a route for the app
+app.get('/', (req, res) => {
+  res.send('Hello World');
+});
+
+// create a route for the /info endpoint
+app.get('/info', (req, res) => {
+
+  var jsonOutputText = '{ "serverName": "' + packageData.name + '", "serverVersion": "' + packageData.version + '" }';
```

Comment:
What do you think about, 
```
{ serverName: ..., 
  serverVersion: ...
}
```
Then you don't need to parse it also

Reply:
If you mean directly returning the string without parsing, Done!